torch/_torch_docs.py

```diff
@@ -8099,105 +8099,81 @@ def merge_dicts(*dicts):
 """.format(**common_args))
 
 add_docstr(torch.std, r"""
-std(input, unbiased=True) -> Tensor
+std(input, dim, unbiased, keepdim=False, *, out=None) -> Tensor
 
-Returns the standard-deviation of all elements in the :attr:`input` tensor.
-
-If :attr:`unbiased` is ``False``, then the standard-deviation will be calculated
-via the biased estimator. Otherwise, Bessel's correction will be used.
+If :attr:`unbiased` is ``True``, Bessel's correction will be used.
+Otherwise, the sample deviation is calculated, without any correction.
 
 Args:
     {input}
-    unbiased (bool): whether to use the unbiased estimation or not
+    {dim}
 
-Example::
+Keyword args:
+    unbiased (bool): whether to use Bessel's correction (:math:`\delta N = 1`).
+    {keepdim}
+    {out}
 
-    >>> a = torch.randn(1, 3)
-    >>> a
-    tensor([[-0.8166, -1.3802, -0.3560]])
-    >>> torch.std(a)
-    tensor(0.5130)
 
-.. function:: std(input, dim, unbiased=True, keepdim=False, *, out=None) -> Tensor
+.. function:: std(input, unbiased) -> Tensor
+   :noindex:
 
-Returns the standard-deviation of each row of the :attr:`input` tensor in the
-dimension :attr:`dim`. If :attr:`dim` is a list of dimensions,
-reduce over all of them.
+Calculates the standard deviation of all elements in the :attr:`input` tensor.
 
-{keepdim_details}
-
-If :attr:`unbiased` is ``False``, then the standard-deviation will be calculated
-via the biased estimator. Otherwise, Bessel's correction will be used.
+If :attr:`unbiased` is ``True``, Bessel's correction will be used.
+Otherwise, the sample deviation is calculated, without any correction.
 
 Args:
     {input}
-    {dim}
-    unbiased (bool): whether to use the unbiased estimation or not
-    {keepdim}
-
-Keyword args:
-    {out}
+    unbiased (bool): whether to use Bessel's correction (:math:`\delta N = 1`).
 
 Example::
 
-    >>> a = torch.randn(4, 4)
-    >>> a
-    tensor([[ 0.2035,  1.2959,  1.8101, -0.4644],
-            [ 1.5027, -0.3270,  0.5905,  0.6538],
-            [-1.5745,  1.3330, -0.5596, -0.6548],
-            [ 0.1264, -0.5080,  1.6420,  0.1992]])
-    >>> torch.std(a, dim=1)
-    tensor([ 1.0311,  0.7477,  1.2204,  0.9087])
+    >>> a = torch.tensor([[-0.8166, -1.3802, -0.3560]])
+    >>> torch.std(a, unbiased=False)
+    tensor(0.4188)
 """.format(**multi_dim_common))
 
 add_docstr(torch.std_mean,
            r"""
-std_mean(input, unbiased=True) -> (Tensor, Tensor)
+std_mean(input, dim, unbiased, keepdim=False, *, out=None) -> (Tensor, Tensor)
```

Comment:
So the idea is that we're going to hide the correction kwarg in the documentation for the next couple weeks?

Reply:
Yes, until the "forward compatibility" period is over.